components/server/src/workspace/workspace-starter.ts

```diff
@@ -374,22 +373,6 @@ export class WorkspaceStarter {
         await client.stopWorkspace(ctx, req);
     }
 
-    protected async checkBlockedRepository(user: User, contextURL: string) {
-        const hit = this.config.blockedRepositories.find((r) => !!contextURL && r.urlRegExp.test(contextURL));
-        if (!hit) {
-            return;
-        }
-        if (hit.blockUser) {
-            try {
-                await this.userService.blockUser(user.id, true);
-                log.info({ userId: user.id }, "Blocked user.", { contextURL });
-            } catch (error) {
-                log.error({ userId: user.id }, "Failed to block user.", error, { contextURL });
-            }
-        }
-        throw new Error(`${contextURL} is blocklisted on Gitpod.`);
-    }
-
     protected async checkBlockedRepositoryInDB(user: User, contextURL: string) {
```

Comment:
Can this be renamed now to just `checkBlockedRepository`? It only had the `InDB` suffix to distinguish it from the method that we've just deleted.